.github/ISSUE_TEMPLATE/tidb-binlog-support.md

```diff
@@ -0,0 +1,28 @@
+---
+title: tidb-binlog support
+category: tools
+---
+
+# TiDB-Binlog Support
+
+Before create a issue, please read this document first:
```

Comment:
lightning issue template is a good example.  you can learn from it @WangXiangUSTC